### PR TITLE
create wordpress site on container instance

### PR DESCRIPTION
**Pull request by wenwu449**
### Best Practice Checklist
Check these items before submitting a PR... See the Contribution Guide for the full detail: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md 

1. uri's compatible with all clouds (Stack, China, Government)
1. Staged artifacts use _artifactsLocation & _artifactsLocationSasToken
1. Use resourceGroup().location for resource locations
1. Folder names for artifacts (nestedtemplates, scripts, DSC)
1. Use literal values for apiVersion (no variables)
1. Parameter files (GEN-UNIQUE for value generation and no "changemeplease" values
1. $schema and other uris use https
1. Use uniqueString() whenever possible to generate names for resources.  While this is not required, it's one of the most common failure points in a deployment. 
1. Update the metadata.json with the current date

For details: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/bp-checklist.md

- [X] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

### Changelog

*
*
*

### CODE REVIEW

File: 201-aci-wordpress/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Create a WordPress site on Container Instance
+
+Create a WordPress site (including its MySQL database) on Container Instance - <a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F201-aci-wordpress%2Fazuredeploy.json" target="_blank">
```
Comment (bmoore-msft) [inline 201-aci-wordpress/README.md]: fix typos, etc as in the other sample

File: 201-aci-wordpress/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+        "shareContainerGroupName": "[concat(resourceGroup().name, '-createshare')]",
+        "wordpressContainerGroupName": "[concat(resourceGroup().name, '-wordpress')]",
+        "wordpressShareName": "[concat(toLower(resourceGroup().name), '-wp')]",
+        "mysqlShareName": "[concat(toLower(resourceGroup().name), '-sql')]"
+    },
+    "resources": [
+        {
+            "type": "Microsoft.Storage/storageAccounts",
+            "name": "[parameters('storageAccountName')]",
+            "apiVersion": "2016-01-01",
```
Comment (bmoore-msft) [inline 201-aci-wordpress/azuredeploy.json]: update apiVersion

File: 201-aci-wordpress/azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+                                "storage",
+                                "share",
+                                "create",
+                                "--name",
+                                "[variables('wordpressShareName')]"
+                            ],
+                            "environmentVariables": [
+                                {
+                                    "name": "AZURE_STORAGE_KEY",
+                                    "value": "[listKeys(parameters('storageAccountName'),'2016-01-01').keys[0].value]"
```
Comment (bmoore-msft) [inline 201-aci-wordpress/azuredeploy.json]: update apiVersion

File: 201-aci-wordpress/azuredeploy.json [hunk 4]
```diff
@@ -0,0 +1,247 @@
...
+                                }
+                            }
+                        }
+                    }
+                ],
+                "volumes": [
+                    {
+                        "azureFile": {
+                            "shareName": "[variables('wordpressShareName')]",
+                            "storageAccountKey": "[listKeys(parameters('storageAccountName'),'2016-01-01').keys[0].value]",
```
Comment (bmoore-msft) [inline 201-aci-wordpress/azuredeploy.json]: update apiVersion

File: 201-aci-wordpress/azuredeploy.json [hunk 5]
```diff
@@ -0,0 +1,247 @@
...
+                            "storageAccountName": "[parameters('storageAccountName')]"
+                        },
+                        "name": "wordpressfile"
+                    },
+                    {
+                        "azureFile": {
+                            "shareName": "[variables('mysqlShareName')]",
+                            "storageAccountKey": "[listKeys(parameters('storageAccountName'),'2016-01-01').keys[0].value]",
```
Comment (bmoore-msft) [inline 201-aci-wordpress/azuredeploy.json]: update apiVersion

File: 201-aci-wordpress/azuredeploy.json [hunk 6]
```diff
@@ -0,0 +1,247 @@
...
+            "metadata": {
+                "description": "MySQL database password"
+            }
+        }
+    },
+    "variables": {
+        "image": "microsoft/azure-cli",
+        "cpuCores": "0.5",
+        "memoryInGb": "0.7",
+        "shareContainerGroupName": "[concat(resourceGroup().name, '-createshare')]",
```
Comment (bmoore-msft) [inline 201-aci-wordpress/azuredeploy.json]: resourceGroup().name can start with non-alphnumeric chars - is that true for all these variables?

File: 201-aci-wordpress/azuredeploy.json [hunk 7]
```diff
@@ -0,0 +1,247 @@
...
+                                "storage",
+                                "share",
+                                "create",
+                                "--name",
+                                "[variables('mysqlShareName')]"
+                            ],
+                            "environmentVariables": [
+                                {
+                                    "name": "AZURE_STORAGE_KEY",
+                                    "value": "[listKeys(parameters('storageAccountName'),'2016-01-01').keys[0].value]"
```
Comment (bmoore-msft) [inline 201-aci-wordpress/azuredeploy.json]: update apiVersion

File: 201-aci-wordpress/azuredeploy.parameters.json [hunk 8]
```diff
@@ -0,0 +1,9 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "mysqlPassword": {
+      "value": "mysql!password"
```
Comment (bmoore-msft) [inline 201-aci-wordpress/azuredeploy.parameters.json]: use GEN-PASSWORD

File: 201-aci-wordpress/azuredeploy.json (hunk 6 above)
Comment (wenwu449) [reply 201-aci-wordpress/azuredeploy.json]: use fixed value since they do not need to be globally unique

**Pull request merged**